Source/Scene/Batched3DModel3DTileContent.js

```diff
@@ -57,11 +57,11 @@ define([
          * The following properties are part of the {@link Cesium3DTileContent} interface.
          */
         this.state = Cesium3DTileContentState.UNLOADED;
-        this.contentReadyToProcessPromise = when.defer();
-        this.readyPromise = when.defer();
         this.batchTableResources = undefined;
         this.featurePropertiesDirty = false;
 
+        this._contentReadyToProcessPromise = when.defer();
+        this._readyPromise = when.defer();
```

Comment:
Up to you, but I would recommend changing `_readyPromise` to `_readyDeferred` and `contentReadyToProcessPromise` to `contentReadyToProcessDeferred` throughout this PR to avoid confusion in the future and make it clear that these are deferreds and not promises.